code/modules/underworld/underworld.dm

```diff
@@ -169,20 +160,61 @@
 	else
 		to_chat(user, "<B><font size=3 color=red>It's LOCKED.</font></B>")
 
+
+/obj/structure/underworld/coinspawner
+	name = "The Hand"
+	desc = "A coin?"
+	icon = 'icons/roguetown/underworld/enigma_husks.dmi'
+	icon_state = "the_hand_c"
+	anchored = TRUE
+	resistance_flags = INDESTRUCTIBLE
+	var/has_coin = TRUE
+
+/obj/structure/underworld/coinspawner/attack_hand(mob/user)
+	. = ..()
+	if(has_coin && isroguespirit(user))
+		on_activation(user)
+
+/obj/structure/underworld/coinspawner/Crossed(atom/movable/AM)
+	. = ..()
+	if(has_coin && isroguespirit(AM))
+		on_activation(AM)
+
+/obj/structure/underworld/coinspawner/proc/on_activation(mob/living/carbon/spirit/fool)
+	var/obj/item/underworld/coin/toll = new(get_turf(src))
+	if(!GLOB.underworld_coinpull_locs.len)
+		if(fool.put_in_hands(toll))
+			to_chat(fool,span_notice("\The [src] puts \the [toll] in your hand..."))
+		else
+			to_chat(fool,span_notice("\The [src] drops \the [toll]..."))
+			toll.forceMove(get_turf(fool))
+		set_coin_taken()
+		return
+	var/turf/moveloc = pick(GLOB.underworld_coinpull_locs)
+	fool.forceMove(moveloc)
+	if(fool.put_in_hands(toll))
+		to_chat(fool,span_alertwarning("\The [src] swiftly drags you under; but leaves you with \the [toll]!"))
+	else
+		to_chat(fool,span_alertwarning("\The [src] swiftly drags you under; but leaves \the [toll] at your feet!"))
+	set_coin_taken()
+
+/obj/structure/underworld/coinspawner/proc/set_coin_taken()
+	has_coin = FALSE
+	icon_state = "the_hand"
+	desc = "A hand?"
+	addtimer(CALLBACK(src,TYPE_PROC_REF(/obj/structure/underworld/coinspawner,regenerate_coin)),5 MINUTES)
```

Comment:
way to short, the reason they don't respawn is we don't want players flooding the game later in the round when things are dilluted. I don't want them respawning.

Reply:
Will bump to 20 minutes

Reply:
The concept of them respawning is the issue, it means the flow of people will almost always be the same which we do not want as it trivializes death, as said remove this

Reply:
The main problem with them *not* respawning is that people just get stuck in underworld waiting for 9 years until 10 min coin pity, this allows the adventurous ones to venture into the maze and *hopefully* find a coin; should arena be busy or the *pit* broken, there are only a few coin spawners

Reply:
Removed.